ecosystem/sep-0038.md

```diff
@@ -35,6 +39,49 @@ All endpoints accept in requests the following `Content-Type`:
 All endpoints respond with content type:
 - `application/json`
 
+### Errors
+
+If an error occurs when calling any endpoint, an appropriate HTTP status code
+will be returned along with an error response.
+
+#### Status Code
+
+Common HTTP status codes may be returned for a server. In particular the following are expected:
+
+Status Code | Name | Reason
+-----|------|------------
+`400` | Bad Request | The request is invalid in any way.
+`403` | Permission Denied | No `Authorization` header has been provided or the contents of the header are not accepted as valid.
+
+The response body must contain a human-readable description of the reason for error:
+
+```json
+{
+  "error": "The requested asset is not supported. See GET /prices for supported assets."
+}
+```
+
+### Asset Identification Format
+
+This protocol can be used to provide quotes for any class of asset in exchange for a Stellar asset. The following format is used to identify an asset in API requests and responses.
+
+```
+namespace:symbol:issuer
+```
+
+The currently accepted `namespace` values are:
+
+Name | Description
+-----|------------
+`stellar` | Used for Stellar assets.
+[`iso4217`](https://www.iso.org/iso-4217-currency-codes.html) | Used for fiat currencies.
+
+The `symbol` value should be the Stellar `asset_code` or ISO-4217 currency code.
+
+The `issuer` value is optional because it may not be relevant for a given asset class, such as fiat currencies. For Stellar assets, `issuer` should be the public key of the asset's issuing account.
+
+When presenting assets to end users, applications should replace the Stellar public key string with the domain hosting the issuer's [SEP-1 `stellar.toml`](sep-0001.md).
```

Comment:
❗  I think this paragraph is out-of-scope for SEP-38. This SEP is concerned with the API between products and shouldn't dictate the UI of user facing applications. It should be something we leave for SEP-32 to recommend. Also because this one sentence is an entire specification on its own that needs fleshing out, and that is essentially SEP-32. Thoughts?